test/parallel/test-child-process-flush-stdio.js

```diff
@@ -8,10 +8,22 @@ const p = cp.spawn('echo');
 p.on('close', common.mustCall(function(code, signal) {
   assert.strictEqual(code, 0);
   assert.strictEqual(signal, null);
+  spawnWithReadable();
 }));
 
 p.stdout.read();
 
-setTimeout(function() {
-  p.kill();
-}, 100);
+function spawnWithReadable() {
+  const buffer = [];
+  const p = cp.spawn('echo', ['123']);
+  p.on('close', common.mustCall(function(code, signal) {
+    assert.strictEqual(code, 0);
+    assert.strictEqual(signal, null);
+    assert.strictEqual(Buffer.concat(buffer).toString(), '123\n');
```

Comment:
Maybe `.trim()` the result?  I'd expect this to fail when the line terminator is `\r\n`.